build_compilers/lib/src/dev_compiler_bootstrap_builder.dart

```diff
@@ -0,0 +1,272 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:convert';
+
+import 'package:analyzer/analyzer.dart';
+import 'package:build/build.dart';
+import 'package:path/path.dart' as _p; // ignore: library_prefixes
+
+import 'dev_compiler_builder.dart';
+import 'module_builder.dart';
+import 'modules.dart';
+
+/// Alias `_p.url` to `p`.
+_p.Context get p => _p.url;
+
+final String bootstrapJsExtension = '.dart.bootstrap.js';
```

Comment:
nit: a bit funky that these start with a leading dot.

Reply:
Ya that is really just because of how `changeExtension` on `AssetId` works, it won't put in the leading dot.

Reply:
nm. I see existing code in this script using the same style so no need to change.